### PR TITLE
Updated for sourcing the TLS Certificate.

### PR DESCRIPTION
**Pull request by sudhirappaji**
Updated the mechanism for sourcing the TLS certificate.

### CODE REVIEW

**Comment by sudhirappaji** [comment]
@arvindn05 @emanoelxavier : Please review

File: designs/osc_containerization/OSC-containerization.md [hunk 1]
```diff
@@ -70,6 +70,16 @@ Example Command to start a docker OSC container with environment variables is:
...
+
+>The OpenDayLight (ODL) uses Karaf distribution for sourcing the TLS certificates (based on the link https://wiki.opendaylight.org/view/OpenDaylight_OpenFlow_Plugin:_TLS_Support). 
+In this solution, various TLS certificates are generated in the relevant format and stored in specific “PATH” as described in the configuration files (*.xml). The configuration files contains various information about the certificates, location of these certification and security algorithms used etc.
+As part of ODL installation, Karaf distribution reads these configuration files and source the new certificates to ODL, so that new certificates gets installed / copied in ODL and used in the new session. All of these are automated as part of Karaf custom command set defined as part of ODL definition.
+
+The sourcing of OSC certificates shall follow the similar scheme,but without using the Karaf Distribution. The OSC certificates like internal certificate (containing internal key and public key), other public certificates (used for authenticating with security manager, openstack clients etc) are sourced as a separate *.p12/*.jks certificates and sourced in a specific folder.
```
Comment (arvindn05) [inline designs/osc_containerization/OSC-containerization.md]: containing ~~internal~~ **private** key and public key.

Can you add more details on the exact files that OSC uses for storing the certificates? Currently OSC has 2 keystores i believe.

File: designs/osc_containerization/OSC-containerization.md [hunk 2]
```diff
@@ -70,6 +70,16 @@ Example Command to start a docker OSC container with environment variables is:
...
+As part of OSC deployment / installation, the relevant deployment scripts shall pick up the new certificates and translates this to the one single truststore and sourced to OSC.
```
Comment (arvindn05) [inline designs/osc_containerization/OSC-containerization.md]: can you add details on how the new certificates are picked up? 

Are we going with the certificates will be stored in jks by default and this jks is provided to OSC for its use? can we add detail on how the password for this JKS file will be provided to OSC?

**Comment by arvindn05** [comment]
can we make this a seperate spec? Its not really related to containerization.

**Comment by sudhirappaji** [comment]
shall be covered in different document

**Pull request closed without merging**